generate/sam-2016-10-31.json

```diff
@@ -752,7 +752,7 @@
                 "Authorizers": {
                     "Documentation": "https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#api-auth-object",
                     "Required": false,
-                    "Type": "String",
+                    "PrimitiveType": "Json",
```

Comment:
I think the type is actually more defined than `Json`: 

https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#api-auth-object

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-authorizer.html

Reply:
Yeah, Json  in this location is a placeholder for a more complicated object (beyond my knowledge) to be constructed.